src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss

```diff
@@ -16,18 +16,19 @@ $blue: #007bff !default;
 $indigo: #6610f2 !default;
 $purple: #6f42c1 !default;
 $pink: #a94442 !default;
-$red: #ef5c55 !default;
+$red: #dc3545 !default;
 $orange: #fd7e14 !default;
-$yellow: #ffc200 !default;
-$green: #0b0 !default;
+$yellow: #d48200 !default;
+$green: #00a800 !default;
+$green-dark: #198754 !default;
```

Comment:
@nSedrickm based on [this page](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=008A00) `#008A00` (vs `#00BB00`) is enough for AA, so why would we want to go for anything darker (`#198754`)?

Reply:
Hi @epuertat. #00BB00 doesnt work well for smaller text  so we needed a much darker one. An example is the pools page(PG status column) 
![pool table](https://user-images.githubusercontent.com/15856785/190641777-eee31484-5ac9-420a-ac52-ab1a9f628fa5.png)

Reply:
> Hi @epuertat. #00BB00 doesnt work well for smaller text so we needed a much darker one. An example is the pools page(PG status column) ![pool table](https://user-images.githubusercontent.com/15856785/190641777-eee31484-5ac9-420a-ac52-ab1a9f628fa5.png)

@nSedrickm now I understood it!: this green color needs to be WCAG-AA compliant both as a background color for labels and foreground color for the PG status, right? Ok so then we could either try:
* Come up with 2 green color variables: `$green-text` and `$green-background`?
* Convert green texts (like the PG status) into labels, so that we only have `$green-background` elements.

What do you think?

Reply:
@epuertat this could do nicely, thanks.  I just made some updates here. The new color `#008A00` you shared earlier is level AA compliant and meets both requirements for the labels and badges.

Here is how the badges are now. If they look ok, we won't need another color. What do you think?

![image](https://user-images.githubusercontent.com/15856785/194928890-ec7ef450-b199-4b0a-80c4-34947f12eeca.png)